src/Components/Books.js

```diff
@@ -0,0 +1,39 @@
+import Form from './Form';
+
+const Books = () => (
+  <div className="container">
+    <div className="container mt-4 border bg-white rounded shadow p-4 ">
+      <div className="row">
+        <div className="col-6">
+          <small className="text-secondary fw-bold">Action</small>
+          <h5 className="fw-bold">
+            The Hunger Game
+            <h6 className="text-primary p-0 m-0">Suzanne Collins</h6>
+          </h5>
+          <ul className="list-inline d-flex pt-2">
+            <li className="text-primary pe-2">Comment</li>
+            <div className="vr" />
+            <li className="text-primary px-2 ">Remove</li>
+            <div className="vr" />
+            <li className="text-primary ps-2 ">Edit</li>
+          </ul>
+        </div>
+        <div className="col-3" />
+        <div className="col-3 border-start">
```

Comment:
- [ ] Please, consider the requirements of this milestone:

- `Renders a list of books using the Book component`

It means that your Books component should be the Book component. 

The Book component should be, as it requered:

- Takes the book data as a prop and renders the title and the author
- Has a button for removing books (the button wouldn't remove the books *yet*, this is just a placeholder for now)

So, as a result, you have to create a Book component separately and use it inside of Books one.